tt_metal/include/compute_kernel_api.h

```diff
@@ -533,6 +533,31 @@ ALWI void heaviside_tile_init() {
     MATH(( llk_math_eltwise_unary_sfpu_heaviside_init<APPROX>() ));
 }
 
+//unary ne : if x !=value --> 1.0, else 0.0
+/**
+ * Performs element-wise computation of:  result = 0 if x!=value , where x is each element of a tile
```

Comment:
First comment says output is 1 if x != value,
next comment says result = 0 if x!=value